files/templates/mbta/account/messages/messages_en.properties

```diff
@@ -1,3 +1,11 @@
+locale_en=English
+locale_es=Español (Spanish)
+locale_ht=Kreyòl Ayisyen (Haitian Creole)
+locale_pt=Português (Portuguese)
+locale_vi=Tiếng Việt (Vietnamese)
+locale_zh-CN=中文简体 (Chinese Simplified)
+locale_zh-TW=中文繁體 (Chinese Traditional)
```

Comment:
these are underscores but elsewhere we use a dash, which is correct? Or are they different in different places?

Reply:
For the language codes, the standard is `-` between segments of the language code. Keycloak expects `locale_<language code>` as the key here.

Reply:
ah yes I misread

Reply:
Question (non-blocking): Is it worth adding a code comment to the effect of `Keycloak expects locale_<language code> as the key`?